.github/tools/cioutput.py

```
@@ -0,0 +1,666 @@
+		use_colors = hasattr(sys.stdout, 'isatty') and sys.stdout.isatty()
+		is_boundary = getattr(record, "is_boundary", False)
+		# Format the basic message
+		message = super().format(record)
+		if is_boundary:
+			return f"[START] {message}" if message else "[ END ]"
+		if use_colors:
+			# Get the color code for this level
+			color = self.COLORS.get(record.levelno, self.RESET)
+			level_name = logging.getLevelName(record.levelno)
+			return f"{color}[{level_name}]{self.RESET} {message}"
+		else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)